.azure-pipelines/test_plan.py

```diff
@@ -130,11 +130,16 @@ def _get_token(self, testbed_tools_url):
         headers = {
             "Content-Type": "application/x-www-form-urlencoded"
         }
+        if testbed_tools_url == "http://sonic-testbed2-scheduler-backend.azurewebsites.net":
+            scope = "api://sonic-testbed-tools-prod/.default"
+        else:
+            scope = "api://sonic-testbed-tools-dev/.default"
+
```

Comment:
Here can simplify as:
```
        scope = "api://sonic-testbed-tools-dev/.default"
        if testbed_tools_url == "http://sonic-testbed2-scheduler-backend.azurewebsites.net":
            scope = "api://sonic-testbed-tools-prod/.default"
```
I think declaring variable `scope` outside of `if-else` is more clear.

Reply:
It's your code originally, I was just splitting them into lines to pass the pre-commit-check...
(BTW, please always make sure your PR and pass the pre-commit-check)
I partly agree, so I refined it by extracting a method.
The best way should be setting the scope in the library, but since it's almost stable and I want to reduce the parameters in the library to reduce the complexity, an extracted method should be OK to both cover the source of scope and clear enough